OpenChart/src/Formats/Osu/Data/OsuBeatmapMetadata.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Generic;
+
+namespace OpenChart.Formats.Osu.Data
+{
+    public class OsuBeatmapMetadata
+    {
+        /// <summary>
+        /// The romanized song title.
+        /// </summary>
+        public string Title { get; set; }
+        
+        /// <summary>
+        /// The song title, containing Unicode characters.
+        /// </summary>
+        public string TitleUnicode { get; set; }
+        
+        /// <summary>
+        /// The romanized song artist.
+        /// </summary>
+        public string Artist { get; set; }
+        
+        /// <summary>
+        /// The song artist, containing Unicode characters.
+        /// </summary>
+        public string ArtistUnicode { get; set; }
+        
+        /// <summary>
+        /// The username of the beatmapper.
+        /// </summary>
+        public string Creator { get; set; }
+        
+        /// <summary>
+        /// The name of this difficulty.
+        /// </summary>
+        public string Version { get; set; }
```

Comment:
Can we rename this to something more intuitive like `Name`?

Reply:
I left it this way because it mirrors the name of the field in the beatmap metadata; the `.osu` file has a line `Version=Hard` for example which corresponds to this field